server/common/error.handler.ts

```diff
@@ -1,14 +1,14 @@
 import { Request, Response, NextFunction } from 'express';
-interface CustomError extends Error {
-  status?: number;
-  errors?: { message: string }[];
-}
+import { HTTPError } from '../todo-api/types';
+import l from './logger';
```

Comment:
No es recomendable usar variables o imports de una sola letra, ya que es dificil de leer. Siempre debemos hacer el codigo para que sea legible para otras personas.
```suggestion
import log from './logger';
```